packages/gatsby/src/bootstrap/cache/__tests__/plugin-hash.js

```diff
@@ -0,0 +1,143 @@
+jest.mock(`md5-file/promise`, () => {
+  const path = jest.requireActual(`path`)
+  return jest.fn(file => Promise.resolve(path.basename(file)))
+})
+const md5File = require(`md5-file/promise`)
+const getPluginHash = require(`../plugin-hash`)
+
+const getHash = ({
+  directory = __dirname,
+  plugins = [],
+  existing = {},
+  ...rest
+} = {}) => getPluginHash({ directory, plugins, existing, ...rest })
```

Comment:
this uses /bootstrap/cache/plugin-hash but methode signature match the function from `packages/gatsby/src/bootstrap/get-plugin-hash.js`

Reply:
this is also resolved with b81e673